src/rules/stories-components.js

```diff
@@ -3,8 +3,14 @@
 const fs = require('fs');
 const path = require('path');
 
-const PATH_REGEX = /\/src\/components\/([^/]+?)\/(presentation|index).(js|ts)$/i;
-const EXPECTED_FILENAMES = ['stories.js', 'stories/index.js', 'stories.ts', 'stories/index.ts'];
+const PATH_REGEX = /\/components\/([^/]+?)\/(presentation|index).(js|ts)$/i;
```

Comment:
wouldn't this path change cause potential issues with the mobile repo? should we have a dedicated file for web with a separate configuration for this rule?

Reply:
This isn't a problem for mobile repo (worker and biz app)
<img width="1044" alt="Screenshot 2023-09-25 at 4 01 01 PM" src="https://github.com/Instawork/eslint-plugin-instawork/assets/2883006/bbd370fc-3c30-480c-b1b3-39e4adfd5293">
<img width="1272" alt="Screenshot 2023-09-25 at 4 00 55 PM" src="https://github.com/Instawork/eslint-plugin-instawork/assets/2883006/39ab5a0a-6539-4380-a29a-174d0e7680b1">

It does flag missing stories for hyperview `src/core/components` because of the pattern change. We can add an override for it in hyperview repo itself. 

I feel the rules should be common as far as possible so that we avoid duplicating. And less rigid in terms of folder pattern matching so that it can still apply if we change the structure slightly. This already happened with the web frontend. 

We can always override in the consuming repo when we need to?